app/plugin_api_test.go

```diff
@@ -620,6 +619,33 @@ func TestPluginAPIRemoveTeamIcon(t *testing.T) {
 	require.Nil(t, err)
 }
 
+func TestPluginAPIUpdateUserActive(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+	api := th.SetupPluginAPI()
+
+	err := api.UpdateUserActive(th.BasicUser.Id, true)
+	require.Nil(t, err)
+	user, err := api.GetUser(th.BasicUser.Id)
+	require.Nil(t, err)
+	require.Equal(t, int64(0), user.DeleteAt)
+
+	err = api.UpdateUserActive(th.BasicUser.Id, false)
+	require.Nil(t, err)
+	if user, err = api.GetUser(th.BasicUser.Id); err != nil {
+		require.Equal(t, "nfke7th9e3bqmn1cg8d1f61o3r", err.Id)
```

Comment:
@lieut-data 635 and 636 are still unchanged as per your previous CR. Let me know if this works out fine.